shell/platform/darwin/ios/BUILD.gn

```diff
@@ -305,21 +305,26 @@ copy("copy_license") {
   outputs = [ "$root_out_dir/LICENSE" ]
 }
 
-shared_library("copy_and_verify_framework_headers") {
+shared_library("copy_and_verify_framework_module") {
+  framework_search_path = rebase_path("$root_out_dir")
   visibility = [ ":*" ]
-  include_dirs = [ "$_flutter_framework_headers_copy_dir" ]
+  cflags_objc = [
+    "-F$framework_search_path",
+    "-fmodules",
+    "-Wnon-modular-include-in-framework-module",
+  ]
 
   sources = [ "framework/Source/FlutterUmbrellaImport.m" ]
   deps = [ ":copy_framework_headers" ]
 }
 
 group("flutter_framework") {
   deps = [
-    ":copy_and_verify_framework_headers",
+    ":copy_framework_module_map",
```

Comment:
The modulemap needs to be copied before the module can be imported.  I don't think the Info.plist is necessary but I vaguely remembering seeing issues when it was missing, so copy that as well since it can't hurt.